src/qibolab/instruments/zhinst/constants.py

```diff
@@ -0,0 +1,4 @@
+"""Shared constants."""
+
+SAMPLING_RATE = 2
+NANO_TO_SECONDS = 1e-9
```

Comment:
Couldn't you restrict the conversion to happen in a single place?

Reply:
not for now. This in fact is not a new file, it is the old utils.py that I renamed, because all other things got removed from it. Not sure why git didn't realize this

Reply:
> So we are indeed getting closer to getting rid of this file :D

Great :)

> This in fact is not a new file, it is the old utils.py that I renamed, because all other things got removed from it. Not sure why git didn't realize this.

The file changed too much in a single commit, including the name. And Git was not able to [trace it was the same file](https://en.wikipedia.org/wiki/Ship_of_Theseus).
But there is still the overall diff. I realized, no worries ^^

Just an example for `NANO_TO_SECONDS` [*].

The reason why it is used in places other than `executor.py` is to convert pulses and sweepers values as well.
However, all these functions are called in `executor.py` and only there, so there is a single interface where pulses and sweepers are received (and in general *exchanged*), i.e. `Zurich.play()`. Couldn't you convert at that level?

This is definitely not required for this PR. But if you're not going to do it here, just open an issue for that.
The goal is not much getting rid of a file, but isolate more of this information, in order to have more modular components (disentangling the web of connections, and reducing possible conversion's mistakes).

[*]: which is the ubiquitous one, while `SAMPLING_RATE` is used only once outside `executor.py`